upup/models/cloudup/resources/addons/digitalocean-cloud-controller.addons.k8s.io/k8s-1.8.yaml.template

```diff
@@ -30,6 +30,7 @@ spec:
       serviceAccountName: cloud-controller-manager
       dnsPolicy: Default
       hostNetwork: true
+      priorityClassName: system-node-critical
```

Comment:
According to [this table](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/), `priorityClassName` was added by default in Kubernetes 1.11. Since this manifest and the `networking.amazon-vpc-routed-eni` one below match older versions of kubernetes, we may need to fork the manifests for k8s 1.11.

Reply:
The functions in [kubemanifest](https://github.com/kubernetes/kops/blob/master/pkg/kubemanifest/priority.go) unconditionally set these and are called without any version checks.

The field is also present in `coredns.addons.k8s.io/k8s-1.6.yaml.template`.

Reply:
My attempt to bring up a 1.8.15 cluster with master kops failed. The API DNS entry never changed from the placeholder address.

Reply:
According to the Kubernetes release notes, `priorityClassName` was added to the pod spec in 1.8.0. So this should not be a blocker.